content/5_Plat/Range_Sweep.mdx

```diff
@@ -139,42 +179,65 @@ template <class T> struct Seg {  // comb(ID,b) = b
 	}
 };
 
+const int MX = 1e5 + 5;
```

Comment:
is +5 necessary

Reply:
Tons of competitive programmers do this lol. This is just in case you want to change up the indexing to 1-indexing or smth without having to deal with out of bound errors. There is 0 reason not to have the +5 even if it is unnecessary.

Reply:
then is your indexing 1-based?

Reply:
No, but it is not like it matters. Just look at the codes here and the "Pro Tip": https://usaco.guide/plat/binary-jump

Reply:
If you want I can explain in the code my reasoning

Reply:
at least cut it down to +1 please
i lowk disagree w that pro tip lol

Reply:
super based pro tip

Reply:
ok i have at least 2 ppl on my side :sunglasses:

Reply:
i think it's good to cut down on the bounds just to clarify to the reader what's being used/not used.

of course it's good practice to avoid bounding errors with +5 in practice/concert, but the code might make more sense to the reader otherwise

i think the decision is up to you -- it's a pretty minor fix though ;)

Reply:
not addressed

Reply:
changed

Reply:
but i am also somewhat confused about the approving process
from what it seems over 50% were in favour of keeping it as it is and aligns with the format of past usaco guide articles

Reply:
no it's just that i'm the one who's more active